solution/ui/regulations/eregs-component-lib/src/components/shared-components/Statutes/table-elements/BodyCell.test.js

```diff
@@ -21,5 +23,22 @@ describe("Statute Table Body Cell", () => {
                 expect(wrapper).toMatchSnapshot();
             });
         });
+
+        it("displays 'None' in Statute Compilation BodyCell when source_url is null", async () => {
+            const statuteCompilationCellSchema = ssaSchema[2];
+
+            const wrapper = render(BodyCell, {
+                props: {
+                    cellData: statuteCompilationCellSchema,
+                    statute: statutesFixture[1],
+                },
+            });
+            await flushPromises();
+
+            const nullCell = wrapper.getByTestId("1302-none").textContent;
+            expect(nullCell).toEqual("None");
```

Comment:
assert against updated mock data that has a `null` `source_url` to make sure it renders "None" instead of a link